simpcli/src/main.rs

```diff
@@ -0,0 +1,112 @@
+// Simplicity "Human-Readable" Language
+// Written in 2023 by
+//   Andrew Poelstra <simplicity@wpsoftware.net>
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the CC0 Public Domain Dedication
+// along with this software.
+// If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+//
+
+use simplicity::human_encoding::Program;
+use simplicity::node::CommitNode;
+use simplicity::{self, BitIter};
+
+use base64::engine::general_purpose::STANDARD;
+use std::env;
+use std::str::FromStr;
+
+/// What set of jets to use in the program.
+// FIXME this should probably be configurable.
+type DefaultJet = simplicity::jet::Elements;
+
+fn usage(process_name: &str) -> Result<(), String> {
+    eprintln!("Usage:");
+    eprintln!("  {} disassemble <base64>", process_name);
+    eprintln!();
+    eprintln!("For commands which take an optional expression, the default value is \"main\".");
+    eprintln!();
+    eprintln!("Run `{} help` to display this message.", process_name);
+    Err("invalid usage".into())
```

Comment:
2459f566bfea051bec2225f82bbdf68a69abfcd1: We should not return an error on `simpcli help`.